src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -56,67 +56,72 @@
 
 namespace
 {
+constexpr int maxTouchPoints = 10; // Current Windows API limitation
+
+// Define constants for tablet API to avoid pulling in an extra Tablet PC header
+const char*              tabletServiceProperty     = "MicrosoftTabletPenServiceProperty";
+const unsigned long long tabletDisablePressAndHold = 0x00000001ULL;
+
 unsigned int               windowCount      = 0; // Windows owned by SFML
 unsigned int               handleCount      = 0; // All window handles
 const wchar_t*             className        = L"SFML_Window";
 sf::priv::WindowImplWin32* fullscreenWindow = nullptr;
+DWORD                      touchIDs[maxTouchPoints];
+POINT                      touches[maxTouchPoints];
```

Comment:
Do we benefit from using `std::array` here? I just want to make sure it's at least considered over a C-style array.

Reply:
Could in theory use a `std::map`, too (internal id as index, hardware id as value). Not sure if that wouldn't be even more clean/readable than using `std::optional`. What do you think?